Documentation/Tools/HolographicRemoting.md

```diff
@@ -115,21 +111,29 @@ You can also temporarily remove the adapter to workaround your issue via the fol
 1. (Optional) Navigate to MRTK / Providers / WindowsMixedReality / Shared / DotNetAdapter in your file explorer (not Unity's Assets view) and delete the `.bin` and `.obj` folders. This removes the local cache of NuGet restored packages for DotNetWinRT.
 1. If you run the MRTK Configurator again, make sure you don't re-enable MSBuild for Unity.
 
+### XR SDK setup instructions
```

Comment:
Do we want to move this next to the legacy header? or maybe decrement the level of the above couple headers to make it clear that they correspond to the legacy path?